### PR TITLE
[17.0] [MIG] l10n_eu_oss_oca

### PR DESCRIPTION
**Pull request by Nikul-OSI**
- [X] account_fiscal_position_partner_type #415

### CODE REVIEW

**Comment by Nikul-OSI** [comment]
@ValentinVinagre @manuelregidor @dreispt Can you please take a look?

**Review by ValentinVinagre — APPROVED** [review]
LGTM 👍🏻

**Comment by dreispt** [comment]
Thank you @Nikul-OSI can you please cherry-pick this fix: https://github.com/OCA/account-fiscal-rule/pull/409

**Review by dreispt — APPROVED** [review]

File: l10n_eu_oss_oca/wizard/l10n_eu_oss_wizard.py [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+        return (
+            eu_country_group.country_ids
+            - self._default_todo_country_ids()
+            - user.company_id.country_id
+        )
+
+    def _default_todo_country_ids(self):
+        user = self.env.user
+        eu_country_group = self._get_eu_res_country_group()
+        eu_fiscal = self.env["account.fiscal.position"].search(
```
Comment (manuelregidor) [inline l10n_eu_oss_oca/wizard/l10n_eu_oss_wizard.py:50]: When an OSS fiscal position is created using the wizard, the oss_oca boolean field is set to True. Is there any reason why that field is not used to search for the already created OSS fiscal positions? Wouldn't it be more clear? Thank you.

Comment (Nikul-OSI) [reply l10n_eu_oss_oca/wizard/l10n_eu_oss_wizard.py:50]: @manuelregidor Yes we can, but here we search the todo_country with country & position type too, That's why.

**Pull request merged**